contrib/storage-mongo/src/test/java/org/apache/drill/exec/store/mongo/TestTableGenerator.java

```diff
@@ -58,7 +59,16 @@ public static void generateTable(String dbName, String collection,
         .jsonArray(jsonArray).importFile(jsonFile).build();
     MongoImportExecutable importExecutable = MongoImportStarter
         .getDefaultInstance().prepare(mongoImportConfig);
-    importExecutable.start();
+    MongoImportProcess importProcess = importExecutable.start();
+
+    try {
+      while (importProcess.isProcessRunning()) {
+        Thread.sleep(1000);
+      }
+    }catch (Exception ex) {
+      logger.error("Import mongoDb failed", ex);
```

Comment:
Here we log the error but go ahead and return. Should we propagate an exception upward in the call stack so that the caller knows that "Something Is Wrong"? Otherwise, how will the caller know whether the import process is ready or not?